arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist

```diff
@@ -943,6 +899,18 @@
 				<key>descriptionText</key>
 				<string>Use different feature request modes to populate the map from a service feature table.</string>
 			</dict>
+			<dict>
+				<key>dependency</key>
+				<array>
+					<string>scottishWildlifeTrust</string>
```

Comment:
My bad, didn't pay enough attention that `LATrailsGeodatabase` and `AuroraCOGeopackage ` are still used in this sample. Also the public art shapefile is no longer used, so can be removed.

I'll fix them in a new PR: https://github.com/Esri/arcgis-runtime-samples-ios/pull/1156